wasm/wazeroir/compiler.go

```diff
@@ -132,15 +132,20 @@ func (c *compiler) resetUnreachable() {
 	c.unreachableState.on = false
 }
 
+type CompilationResult struct {
+	Operations   []Operation
```

Comment:
need to be exported? if so, document. Especially LabelCallers is a little jargon so worth it... also any relationship if any between LabelCallers and Operations (ex if the int value is related to the index or something else)

Reply:
https://github.com/tetratelabs/wazero/pull/60/commits/408b3d8361367f6a96457ba821ca9da041b85220 commented